package.json

```diff
@@ -13,9 +13,27 @@
     "icons"
   ],
   "author": "Chris Jennings <chriskjennings@gmail.com>",
-  "license": "SIL OFL 1.1",
+  "license": "OFL-1.1",
   "bugs": {
     "url": "https://github.com/getsentry/platformicons/issues"
   },
-  "homepage": "https://github.com/getsentry/platformicons"
+  "homepage": "https://github.com/getsentry/platformicons",
+  "scripts": {
+    "build": "rm -rf ./build && tsc",
+    "prepare": "yarn build"
+  },
+  "peerDependencies": {
+    "react": "^16 || ^17",
+    "react-dom": "^16 || ^17"
+  },
+  "devDependencies": {
+    "react": "^16 || ^17",
+    "react-dom": "^16 || ^17",
+    "typescript": "^4.0.2"
+  },
+  "dependencies": {
+    "@types/react": "^16 || ^17",
```

Comment:
are these actually dependencies? honest question as i dont know how this works

Reply:
General consensus is that if you are creating a library and exposing @types, you should put them into dependencies, otherwise devDependencies are fine.

https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html#dependencies
https://github.com/Microsoft/types-publisher/issues/81

There are few gotchas with both approaches (even one where you put them into peerDependencies).